README.rst

```diff
@@ -37,19 +37,42 @@ open-mastr
 Introduction
 ============
 
-The `Marktstammdatenregister (MaStR) <https://www.marktstammdatenregister.de/MaStR>`_ is a German register 
-provided by the German Federal Network Agency (Bundesnetzagentur / BNetza) that keeps track of all power and gas units located in Germany.
+The python package ``open-mastr`` provides an interface for accessing the `Marktstammdatenregister (MaStR) <https://www.marktstammdatenregister.de/MaStR>`_ data. The MaStR is a German register provided by the German Federal Network Agency (Bundesnetzagentur / BNetza) that keeps track of all power and gas units located in Germany. It is a daily growing dataset with more than 8.2 million data points covering electricity and gas production units, electricity and gas consumers, storages, grids, and energy market participants (as of spring 2024).
 
-The MaStR data can be
- 
-#. browsed and filtered `online <https://www.marktstammdatenregister.de/MaStR>`_
-#. taken from `daily provided dumps <https://www.marktstammdatenregister.de/MaStR/Datendownload>`_
-#. be accessed via the `web service <https://www.marktstammdatenregister.de/MaStRHilfe/subpages/webdienst.html>`_
+Independent of ``open-mastr``, the MaStR data can generally be accessed via three main options:
 
-| The python package ``open-mastr`` provides an interface for accessing the data. 
-| It contains methods to download and parse the xml files (bulk) and the SOAP web service (API).
-| In this repository we are developing methods to analyze, validate and enrich the data.
-| We want to collect and compile post processing scripts to improve data quality.
+#. browse, filter and download `in the browser <https://www.marktstammdatenregister.de/MaStR>`_
+#. download `daily provided dumps <https://www.marktstammdatenregister.de/MaStR/Datendownload>`_
+#. access via the `web service <https://www.marktstammdatenregister.de/MaStRHilfe/subpages/webdienst.html>`_
+
+These access options, however, are not exactly frictionless. ``open-mastr`` thus provides an interface for and improved developer experience of accessing the data. This project is intended for individuals who wish to "just work" with the MaStR data and who do not want to deal with the idiosyncrasies of the three access options above.
+
+In particular, ``open-mastr`` facilitates access to the daily provided MaStR dumps with download methods (bulk) and by parsing the XML files to a relational database. Furthermore, the software provides a Python wrapper to access the MaStR SOAP web service (API).
+
+.. note::
```

Comment:
This .. note:: seems to be specific for `.md`
In .rst files it does not render.